dashboard/config/scripts/levels/CSD U2 text style sample.level

```diff
@@ -5,7 +5,7 @@
ummary>**What is a style sheet?**</summary>\r\nA style sheet is a document that controls how a web page will appear.  External style sheets are separate files that are linked to the HTML page.\r\n</details>\r\n<details>\r\n<summary>**What is CSS?**</summary>\r\nCSS is a language that controls the way content on a web site appears.  It uses rule-sets to change the look of a page.  Each rule set has a selector, which specifies which elements on the page will be affected, and the rules, which explain how to display the elements.   Each rule consists of a property and a value for that property.\r\n\r\n\r\nHere’s an example of a rule-set that makes the text in an `h2` tag blue and cursive:\r\n\r\n<pre>h2 {\r\n   color: blue;\r\n   font-family: cursive;\r\n</pre>\r\n\r\nIn this example, `h2` is the selector, `color` and `font-family` are properties, and `blue` and `cursive` are the values for those properties.\r\n</details>\r\n<details>\r\n<summary>**Why is the \"style.css\" file in a different language?**</summary>\r\nWeb developers intentionally separate content and style to make their code easier to maintain. HTML is the language used to structure the content of a web page. CSS is a language that adds style to a web page.  \r\n</details>\r\n<details>\r\n<summary>**What does `px` mean / What is a pixel?**</summary>\r\nPixel, which is abbreviated `px`, is how elements are measured in CSS.  It’s the size of a single point of light on the screen.\r\n</details>\r\n<details>\r\n<summary>**What does the `text-decoration` property do?**</summary>\r\nThe `text-decoration` property puts a line over, under, or through a piece of text.  You can read more about the property at <a href=\"http://www.w3schools.com/cssref/pr_text_text-decoration.asp\">W3Schools - CSS text-decoration Property</a>\r\n</details>\r\n<details>\r\n<summary>**What does the `text-align` property do?**</summary>\r\nThe `text-align` property aligns the text to the left, right, or center. You can read more at <a href=\"http://www.w3schools.com/cssref/pr_text_text-align.asp\">W3Schools - CSS text-align Property</a>\r\n</details>\r\n<details>\r\n<summary>**What does the `font-family` property do?**</summary>\r\nThe `font-family` property changes the font of the text.  It gives a general description of the type of font that should be used, but each browser might display the font slightly differently.  Read more at <a href=\"http://www.w3schools.com/cssref/pr_font_font-family.asp\">W3chools - CSS font-family Property</a>\r\n</details>\r\n</div>\r\n CSS and Style sheets\r\n\r\n* Look at the web page below and find some different types of text stylings that you don't know how to code yet.\r\n* Click on the style sheet (\"style.css\" in the files area) and look at the code inside the file.\r\n* With your partner, discuss which parts of the code you think are making the text appear differently than before.\r\n\r\nIn the next few lessons, you'll be changing and adding new styles on this page.\r\n\r\n",
```

Comment:
intentional to remove the `#` before "CSS and Style sheets"?

Reply:
Nope, that was a mistake!  I'll get it fixed.